misc/showcase.css

```diff
@@ -0,0 +1,740 @@
+  border: 0;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+  color: #dbdada;
+  display: block;
+  height: 22px;
+  padding-bottom: 1px;
+  padding-left: 22px;
+  padding-top: 1px;
+}
+@media (min-width: 768px) {
+  .showcase header .navbar.navbar-default .navbar-form .form-control {
+    width: 100%;
+  }
+}
+@media (min-width: 992px) {
+  .showcase header .navbar.navbar-default .navbar-form .form-control {
+    display: inline-block;
+    width: 130px;
+  }
+}
+.showcase header .navbar.navbar-default .navbar-form .form-control:focus {
+  background: rgba(255, 255, 255, 0.3);
+  color: #fff;
+  -webkit-transition-duration: 400ms;
+  transition-duration: 400ms;
+  -webkit-transition-property: background, color;
+  transition-property: background, color;
+}
+@media (min-width: 768px) and (max-width: 991px) {
+  .showcase header .navbar.navbar-default .navbar-header {
+    float: none;
+    margin-right: -20px;
+    margin-left: -20px;
+  }
+}
+@media (min-width: 768px) and (max-width: 991px) {
+  .showcase header .navbar.navbar-default .navbar-right {
+    float: none !important;
+    margin-right: 0;
+  }
+}
+.showcase header .navbar.navbar-default .navbar-right > li.active > a,
+.showcase header .navbar.navbar-default .navbar-right > li.active:hover > a {
+  background: transparent !important;
+  color: #3bb1dc !important;
+}
+.showcase header .navbar.navbar-default .navbar-right > li:hover {
+  background: #212121;
+}
+.showcase header .navbar.navbar-default .navbar-right > li:hover > a {
+  background: transparent;
+  color: #fff;
+}
+.showcase header .navbar.navbar-default .navbar-right > li > a {
+  color: #eee;
+  padding: 7px 20px;
+}
+@media (min-width: 992px) {
+  .showcase header .navbar.navbar-default .navbar-right > li > a {
+    padding: 25px 15px;
+  }
+}
+.showcase header .navbar.navbar-default .navbar-right > li > a:hover {
+  color: #fff;
+}
+@media (min-width: 992px) {
+  .home .showcase header .navbar.navbar-default .navbar-right > li:hover,
+  .home .showcase header .navbar.navbar-default .navbar-right > li.open {
+    background: #252525;
+    background: rgba(0, 0, 0, 0.1);
+  }
+}
+.showcase header .navbar.navbar-default .navbar-toggle {
+  background-color: transparent;
+  border: none;
+  float: left;
+  margin-bottom: 19px;
+  margin-left: 20px;
+  margin-right: 15px;
+  margin-top: 19px;
+}
+@media (min-width: 768px) and (max-width: 991px) {
+  .showcase header .navbar.navbar-default .navbar-toggle {
+    display: block;
+  }
+}
+.showcase header .navbar.navbar-default .navbar-toggle:focus {
+  outline: none;
+}
+.showcase header .navbar.navbar-default .navbar-toggle:focus .icon-bar,
+.showcase header .navbar.navbar-default .navbar-toggle:hover .icon-bar {
+  background-color: #fff;
+  -webkit-box-shadow: 0 0 3px rgba(255, 255, 255, 0.7);
+  box-shadow: 0 0 3px rgba(255, 255, 255, 0.7);
+}
+@media (min-width: 768px) {
+  .showcase header .navbar-utility .applauncher-pf .dropdown-menu {
+    margin-top: 0;
+  }
+}
```

Comment:
@dabeng you missed one piece of css code here

.dropdown-kebab-pf .dropdown-menu.dropdown-menu-right::after,
.dropdown-kebab-pf .dropdown-menu.dropdown-menu-right::before {
right: 10px;
}